xml/operations-maintenance-upgrade_maintenance.xml

```diff
@@ -0,0 +1,1834 @@
+         the node has already successfully rebooted onto the new kernel, and
+         you just need to get the required services running again on the node.
+       </para>
+       <para>
+         It can sometimes take up to 15 minutes for the <literal>monasca-thresh</literal>
+         service to successfully start in such cases.
+       </para>
+     <step>
+       <para>
+         However, if the service still fails to start after that time, then
+         you may need to force a restart of the <literal>storm-nimbus</literal> and
+         <literal>storm-supervisor</literal> services on all nodes in the
+         <literal>MON-THR</literal> node group, as follows:
+       </para>
+       <screen>&prompt.ardana;cd ~/scratch/ansible/next/ardana/ansible
+&prompt.ardana;ansible MON-THR -b -m shell -a "systemctl restart storm-nimbus"
+ardana-cp-mml-m1 | success | rc=0 >>
+
+
+ardana-cp-mml-m2 | success | rc=0 >>
+
+
+ardana-cp-mml-m3 | success | rc=0 >>
+
+
+ardana > ansible MON-THR -b -m shell -a "systemctl restart storm-supervisor"
+ardana-cp-mml-m1 | success | rc=0 >>
+
+
+ardana-cp-mml-m2 | success | rc=0 >>
+
+
+ardana-cp-mml-m3 | success | rc=0 >>
+
+
+ardana > ansible-playbook -i hosts/verb_hosts _ardana-post-reboot.yml --limit ardana-cp-mml-m1
+</screen>
+     </step>
+   </procedure>
+   <para>
+     If the <literal>monasca-thresh</literal> service still fails to start up,
+     contact your support team.
+   </para>
+
+   <para>
+     To check which control plane nodes have not yet been rebooted onto
+     the new kernel, you can use an Ansible command to run the command <command>uname -r</command>
+     on the target nodes, as follows:
+   </para>
+   <screen>&prompt.ardana;cd ~/scratch/ansible/next/ardana/ansible
+&prompt.ardana;ansible -i hosts/verb_hosts 'resources:!OPS-LM--first-member:!NOV-CMP' -m command -a 'uname -r'
+ardana-cp-dbmqsw-m1 | success | rc=0 >>
+4.12.14-95.57-default
+
+ardana-cp-dbmqsw-m3 | success | rc=0 >>
+4.12.14-95.57-default
+
+ardana-cp-osc-m1 | success | rc=0 >>
+4.12.14-95.57-default
+
+ardana-cp-dbmqsw-m2 | success | rc=0 >>
+4.12.14-95.57-default
+
+ardana-cp-mml-m2 | success | rc=0 >>
+4.12.14-95.57-default
+
+ardana-cp-osc-m2 | success | rc=0 >>
+4.12.14-95.57-default
+
+ardana-cp-mml-m1 | success | rc=0 >>
+4.12.14-95.57-default
+
+ardana-cp-mml-m3 | success | rc=0 >>
+4.12.14-95.57-default
+
+ardana > uname -r
+4.12.14-95.57-default
+   </screen>
+   <para>
+     If any node's <command>uname -r</command> value does not match the kernel
+     that the deployer is running, you probably have not yet rebooted that node.
+   </para>
+
+   <procedure>
+     <title>Rebooting the Compute Nodes</title>
+     <para>
+       Finally, you need to reboot the compute nodes. Rebooting multiple
+       compute nodes at the same time is possible, so long as doing so does
+       not compromise the integrity of running workloads. We recommended
+       that you evacuate groups of compute nodes in a controlled fashion,
```

Comment:
I think I missed this "evacuate" when I was changing to "migrate". Probably should change this line to read "that you migrate workloads off groups of compute nodes in a controlled fashion,"